swap/oracle.go

```diff
@@ -22,7 +22,6 @@ type PriceOracle interface {
 }
 
 // NewPriceOracle returns the actual oracle to be used for discovering the price
-// TODO: Add a config flag so that this can be configured via command line
```

Comment:
Although does the SWIP explicitly describe that the oracle be configurable via config flag?
If not, a separate issue might well be useful.

Reply:
It does not explicitly mention this and I think this is also still up to debate. My personal preference would be to set it in a config file (as it is not just mentioning the oracle, but actually a *preference list for oracles*). Anyways, the SWIP ensures we won't forget this and implementation details will be discussed for sure!

Reply:
OK. Fyi, Swarm supports different levels of configuration. It is possible to config via a config file with a define structure, and then this can be overridden by command line parameters and/or environment variables.

Thus whatever will be done for this will need to comply to this structure, don't invent a separate config file or something.